docs/user-guide/toc.md

```diff
@@ -120,26 +143,17 @@ Through this toolbar it is possible to:
 
 * [Create Widgets](widgets.md) for the selected layer <img src="../img/button/widgets.jpg" class="ms-docbutton"/>
 
-Selecting a group the toolbar is the following:
+* Access the **Compare tool** where it is possible *Swipe* or *Spy* the selected layer <img src="../img/button/compare_tool_button.jpg" class="ms-docbutton"/>. 
 
-<img src="../img/toc/group-toolbar.jpg" class="ms-docimage"/>
+From the dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/> by clicking on <img src="../img/button/swipe_button.jpg" class="ms-docbutton" style="max-height:25px;"/> button a draggable handle appears on the map to swipe the selected layer. 
```

Comment:
From the dropdown menu <img src="../img/button/dropdown_menu_button.jpg" class="ms-docbutton" style="max-height:25px;"/> of the **Compare tool** it is possible to click on <img src="../img/button/swipe_button.jpg" class="ms-docbutton" style="max-height:25px;"/> button so that  the Swipe tool is enabled on the map for the selected layer: to activate the Swipe it is also possible to simply click on the **Compare tool** button.